client/src/components/profilePage/profilePage.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Navbar from "../navbar/Navbar";
+import "./profilePage.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "font-awesome/css/font-awesome.min.css";
+import logo from "../../components/profile-pic.png";
+import Footer from "../Footer/footer";
+import { Link } from "react-router-dom";
+class profilePage extends Component {
+    render() {
+        return (
+            <div>
+                <Navbar />
+
+                <img
+                    src={logo}
+                    className="rounded-circle"
+                    alt="Profile-Picture"
+                />
+                <div className="Intro">
+                    <h1>Purnima Sharma</h1>
+                    <h5>Open Source Project Manager</h5>
+                </div>
+                <div className="Header">
+                    <h3>
+                        {" "}
+                        <Link to="/profile" className="profile">
+                            Profile
+                        </Link>
+                        <span>|</span>
+                        <Link to="/addaquestion">Questions</Link>
+                        <span>|</span> <Link to="feed">Answers</Link>
+                    </h3>
+                    <hr></hr>
+                </div>
+                <div className="followers">
+                    <h3>
+                        {" "}
+                        Following - <span>10</span> &nbsp; &nbsp;
```

Comment:
One suggestion! Make this static value of followers convert into a variable and declare above.

Reply:
According to github followers?

Reply:
No no, Just declare variable in above, so when data comes from backend than that would become easy to find.
ex. `const following = 0`
`Following - <span>{following}</span> &nbsp; &nbsp;`

Reply:
Okay fine both both followers and following?